javascript/javascript1/week4/palindromicSubstring.js

```diff
@@ -0,0 +1,34 @@
+var longestPalindrome = function(string) {
```

Comment:
why are you using `var`? We said many times in class use `const` or `let`

Reply:
again, you shouldn't use the word `string` as a variable name, it's a special keyword denoting the string _type_ . Technically legal, but not a good practice.